cs/commandstation/cm_test_helper.hxx

```diff
@@ -14,6 +14,8 @@ openlcb::MockSNIPUserFile snip_user_file("Default user name",
 const char* const openlcb::SNIP_DYNAMIC_FILENAME =
     openlcb::MockSNIPUserFile::snip_user_file_path;
 
+unsigned spiffsReadCount = 0;
```

Comment:
Should this have an ifdef GTEST wrapper around it?

Reply:
This is the test helper code, so this code only gets compiled in gtest.